src/Motions/Word.ts

```diff
@@ -8,45 +8,56 @@ export class MotionWord extends Motion {
     private wordDelta: MotionWordPosition;
     private useBoundaryIfChange = false;
 
-    static wordSeparators = './\\\\()"\'\\-:,.;<>~!@#$%^&*|+=\\[\\]{}`~?';
+    private static wordSeparators = './\\\\()"\'\\-:,.;<>~!@#$%^&*|+=\\[\\]{}`~?';
+    private static blankSeparators = '\\s';
+    
+    separators: string;
 
-    static nextStartOrBoundaryIfChange(): Motion {
-        const obj = MotionWord.nextStart();
+    constructor(options: {useBlankSeparatedStyle: boolean} = {useBlankSeparatedStyle: false}) {
+        super();
+
+        this.separators = options.useBlankSeparatedStyle
+            ? MotionWord.blankSeparators : MotionWord.wordSeparators;
+    }
+
+    static nextStartOrBoundaryIfChange(args: {useBlankSeparatedStyle: boolean} = {useBlankSeparatedStyle: false}): Motion {
```

Comment:
Shouldn't the TypeScript compiler catch this kind of errors?
When I try you code snippet I get the following TypeScript error:
```
[ts] Argument of type '{ someOtherParam: boolean; }' is not assignable 
to parameter of type '{ useBlankSeparatedStyle: boolean; }'. 
Object literal may only specify known properties, 
and 'someOtherParam' does not exist in type '{ useBlankSeparatedStyle: boolean; }'.
```

But I am new to TypeScript, so I might understand something wrong...